huaweicloud/services/cts/resource_huaweicloud_cts_tracker.go

```diff
@@ -0,0 +1,269 @@
+package cts
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	cts "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/cts/v3/model"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+func ResourceCTSTracker() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCTSTrackerUpdate,
+		ReadContext:   resourceCTSTrackerRead,
+		UpdateContext: resourceCTSTrackerUpdate,
+		DeleteContext: resourceCTSTrackerDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceCTSTrackerImportState,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+
+			"bucket_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"file_prefix": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				RequiredWith: []string{"bucket_name"},
+			},
+			"lts_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"validate_file": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"kms_id": {
```

Comment:
should we need to add `RequiredWith` bucket_name for `kms_id` and `validte_file` as well?

Reply:
fixed